src/controller/tag-controller.js

```diff
@@ -0,0 +1,40 @@
+import TagService from '../service/tag-service.js';
+
+class TagController {
+  async getTags(req, res, next) {
+    try {
+      const { page = 1, limit = 10, order = 'desc', search } = req.query;
```

Comment:
```suggestion
      const page = parseInt(req.query.page || 1, 10);
      const limit = parseInt(req.query.limit || 10, 10);
      const { order = 'desc', search } = req.query;

      if (Number.isNaN(page) || page < 1 || Number.isNaN(limit) || limit < 1) {
        return res.status(400).json({ error: '유효하지 않은 페이지 또는 리밋 값입니다.' });
      }
```
페이지와 리밋에 숫자 값이 들어오는지 유효성 검사를 진행하는 것도 좋을 것 같습니다!